eth/utils/rlp.py

```diff
@@ -11,9 +16,15 @@
     ValidationError,
 )
 
+from eth.rlp.blocks import (
+    BaseBlock,
+)
+
+DiffObjectType = Union[None, Iterable[Tuple[str, str, str]]]
```

Comment:
`Union[None, Something]` should be equivalent to `Optional[Something]` but the latter reads more naturally